### DISCUSSION TITLE
deepjavalibrary/jupyter Docker compatibility with arm64

### DISCUSSION THREAD

**Discussion by r7b7** [deepjavalibrary/djl#3548] [Development]
I'm running deepjavalibrary/jupyter image but it runs with a warning - 
`WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested`

my platform is ARM64 so the warning makes sense, my Q is - is there a way to fix this?

Tried to access the file mentioned here https://docs.djl.ai/master/docs/demos/jupyter/index.html#install-ijava-kernel-for-jupyter but that throws a 404.

I'm a DJL newbie, if this topic is discussed elsewhere please share links. Thanks in advance.

**Comment by frankfliu** [comment]
The link should be: https://github.com/deepjavalibrary/djl-demo/blob/master/jupyter/Dockerfile

@xyang16 can you help fix the broken link?

**Reply by r7b7** [reply]
Thank you!

**Comment by xyang16** [comment]
Raised PR https://github.com/deepjavalibrary/djl-demo/pull/530
